draft-ietf-teep-architecture.md

```diff
@@ -664,8 +661,12 @@ that will be managing the TA in various devices. At step 3, a user
 will go to an Application Store to download the Untrusted
 Application. The Untrusted Application will trigger TA installation
```

Comment:
"The Untrusted Application will trigger" is possible, but I wouldn't say it's a desirable developer experience.
Recommend "Since the Untrusted Application depends on the TA, installing the Untrusted Application will trigger" (which tries to leave it ambiguous whether it's the app or the installer that does the actual trigger).